doc/Overview/Mvux/Advanced/Pagination.md

```diff
@@ -0,0 +1,398 @@
+            await PeopleService.GetPeopleAsync(pageSize: PageSize, pageIndex: pageRequest.Index, ct));
+}
+```
+
+The `AsyncPaginated` method generates a `ListFeed` that supports pagination.
+
+Whenever the user scrolls down to see additional data and is hitting the end of the collection displayed in a `ListView`, the pagination List-Feed is automatically triggered with a page request.  
+The parameter of `AsyncPaginated`, is a delegate taking in a [`PageRequest`](#the-pagerequest-type) value and a `CancellationToken` and returning an `IListFeed<T>` where `T` is `Person` in our case. This delegate is invoked when a page request comes in.
+
+As you can see, we are sending the `Index` property of the incoming `pageRequest` argument to determine what page we're positioned.  
+`PageSize` refers to a constant value of `20`, but the `PageRequest` also has a `DesiredSize` property which the `ListView` can set according to its capacity. So essentially you can substitute `pageSize: PageSize` with `pageSize: pageRequest.DesiredSize` which is a nullable `uint`, and is populated by the `ListView`. We used a constant `PageSize` for the sake of this demonstration.
+
+### View
+
+```xml
+<Page 
+    x:Class="PaginationPeopleApp.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PaginationPeopleApp"
+    xmlns:mvux="using:Uno.Extensions.Reactive.UI">
+
+    <Page.Resources>
+        <DataTemplate x:Key="PersonDataTemplate" x:DataType="local:Person">
+            <StackPanel Orientation="Horizontal" Spacing="2">
+                <TextBlock Text="{x:Bind FirstName}"/>
+                <TextBlock Text="{x:Bind LastName}"/>
+            </StackPanel>
+        </DataTemplate>
+    </Page.Resources>
+
+    <ListView ItemsSource="{Binding PeopleAuto}" ItemTemplate="{StaticResource PersonDataTemplate}"
+        DataFetchSize="20" Header="Next page auto-loading when on available capacity or scrolled"/>
+
+</Page>
+```
+
+In addition, make sure the `DataContext` of the `Page` is set to the generated bindable model called `BindablePeopleModel` (*MainPage.xaml.cs*):
+
+```csharp
+public MainPage()
+{
+    this.InitializeComponent();
+
+    this.DataContext = new BindablePeopleModel(new PeopleService());
+}
+```
+
+> [!TIP]  
+> You can inspect the generated code by either placing the cursor on the word `BindablePeopleModel` and hitting <kbd>F12</kbd>, see other ways to inspect the generated code [here](xref:Overview.Mvux.Advanced.InspectGeneratedCode).
+
+As you can see, there's nothing special in the XAML code as MVUX is taking advantage of the tools already implemented with the `ListView`.
+
+- When the page load, the first 20 items are loaded (after a delay of one second - as simulated in the service).
+- If there's available space in the `ListView`, the next batch of 20 items will be requested from the Feed and loaded from the service thereafter.
```

Comment:
I would encourage the use of the `DesiredSize` instead of relying on a hard coded page size of 20.

Reply:
I chose to write 20 because the default value is 76. Like 4 pages ahead. So the demonstration is a bit lame as it's not evident data is loading.

Reply:
The 76 depends of the size of the `ListView` and the size of the items. For you sample you can just increase the height of your items, so the `ListView` will requests smaller pages.